src/components/Footer/index.tsx

```diff
@@ -202,29 +258,33 @@ const Footer = () => {
             </div>
           </div>
           <div className={styles.Box}>
-            <Text type='subtitle-1' as='h3' className={styles.SectionTitle}>
-              <Translate>We&apos;re here to help</Translate>
-            </Text>
+            <Heading.H5 className={styles.SectionTitle}>
+              <Translate>We're here to help</Translate>
+            </Heading.H5>
             <p className={styles.SectionContent}>
```

Comment:
`<p>` we use to write text not for wrapper/container, use `div` instead if required.